### ISSUE TITLE
Provide a `Console` function that returns the original output so that it can be used in a pipe

### ISSUE THREAD

**Issue by tcoopman** [facebookexperimental/reason-native#115]
It would be very nice if this was possible:

```reasonml
foo
|> List.map(mapper)
|> Console.log()
|> Array.of_list
```

**Comment by kyldvs** [comment]
I wonder how we would want to implement this. If we want to provide implementations for each kind of `Console.log/warn/etc` that would double the number of functions. Perhaps duplicating each function under a `Pipe` module like this would be good though:

```
foo
|> List.map(mapper)
|> Console.Pipe.log()
|> Array.of_list
```

**Comment by kyldvs** [comment]
@bandersongit / @jordwalke do we have documentation for doing new releases? This should be fixed with a new release of Console

**Comment by bandersongit** [comment]
> @bandersongit / @jordwalke do we have documentation for doing new releases? This should be fixed with a new release of Console

I don't think there is documentation, currently it is just manual. Make sure your changes are on master, commit an update to the foo.json file and Changelog.MD and then run ```node scripts/esy-prepublish.js foo.json``` and follow the instructions from there (cd to the generated directory and npm publish).

**Comment by bandersongit** [comment]
This has been released in Console 0.1.0
